web/beacon-app/src/features/members/hooks/useFetchMember.ts

```diff
@@ -17,7 +17,7 @@ export function useFetchMember(memberID: string): MemberQuery {
     ...memberDetailQuery(memberID),
     onError: (error: any) => {
       // stop logging 401 & 403 errors to sentry
-      if (error.response.status !== 401 && error.response.status !== 403) {
+      if (error?.response?.status !== 401 && error?.response?.status !== 403) {
```

Comment:
I am new to typescript and optional chaining - does this mean that if the error is undefined it will be not equal to both 401 and 403 and the exception will still be logged in the line below?

Reply:
Optional chaining, will check if the response object is null or undefined before attempting to read status. Currently, an error is thrown because the response hasn't been returned before attempting to read status.